### PR TITLE
Greet new contributers on first issue/pr

### PR DESCRIPTION
**Pull request by dviererbe**
This GitHub Action will write a comment to the first issue/pr of a new contributor. Aside from just being a welcome message it also reminds people to attend the next `#ubuntu-meeting` and gives instructions how to join the `irc.libera.chat` IRC Server.

### CODE REVIEW

**Comment by dviererbe** [comment]
While fixing the different time formats used as [discussed in the last MIR meeting](https://irclogs.ubuntu.com/2023/08/22/%23ubuntu-meeting.html), I noticed that there is a [standard how to format dates in the Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en#dates), but no standard for how to format time. I [asked the Documentation Team](https://chat.canonical.com/canonical/pl/g9sse8b7jpdfifr7hesfs6kzxc) for recommendations and they suggested ISO 8601. While ISO 8601 is a great standard for machine readability; I strongly disagree that `16:30+01:00` is more user friendly than `16:30 CET` or `16:30 UTC+1`. I commited to `16:30 CET`. I also used the [HTML `<time>` Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time), but it seems that Github removes this tag from markdown :/ I am happy to change it to any other format if you wish.

**TL;DR:** I used the `16:30 CET` time format across all documents. I change it to any other format if you wish.

**Comment by dviererbe** [comment]
Note: I enabled spell checking for the GitHub Action Workflow files (now that we have more user-facing text in them) that are not spell checking config files. It took some trial and error to configure it correctly. Therefore I had to rewrite the history to do some cleanup.

**Comment by dviererbe** [comment]
@cpaelzer this PR is now ready for re-Review. I can't add reviewers to this PR.

**Review by slyon — APPROVED** [review]
Thanks, lgtm.

**Review by didrocks — APPROVED** [review]
LGTM to use CET too. Thanks for the other changes!

**Comment by cpaelzer** [comment]
Thanks, I also consider 16:30 CET much more consumable for human beings.
+1 just like the others and merging ...
